test/run_test.py

```diff
@@ -363,6 +363,16 @@ def print_to_stderr(message):
     print(message, file=sys.stderr)
 
 
+# Convert something like pytorch_windows_vs2019_py36_cuda10.1_build to pytorch_windows_vs2019_py36_cuda10.1
+def get_stripped_CI_job() -> str:
+    job = os.environ.get("CIRCLE_JOB", "").rstrip('0123456789')
+    if job.endswith('_test'):
+        job = job[:len(job) - len('_test')]
+    elif job.endswith('_build'):
+        job = job[:len(job) - len('_build')]
+    return job
```

Comment:
You could instead use regexes here, something like this:
```suggestion
    import re
    job = os.environ.get("CIRCLE_JOB", "")
    m = re.match(r'^(.*?)(?:(?:_test)|(?:_build))?\d*$', job)
    if m:
        job, = m.groups()
    return job
```
(up to you though)

Reply:
Yea I considered using regex's--do you think the added import is worth it? I also thought the current code was easier to read but that might be because I'm not great at regex's.

Reply:
Honestly after writing that regex, I'm not sure it's necessarily an improvement in this case: it uses some tricky things like `*?`, so maybe it's best to just leave the code as-is.